app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -15,6 +15,7 @@ type Mutation {
     setBillingAddressOnCart(input: SetBillingAddressOnCartInput): SetBillingAddressOnCartOutput
     setShippingMethodsOnCart(input: SetShippingMethodsOnCartInput): SetShippingMethodsOnCartOutput @resolver(class: "\\Magento\\QuoteGraphQl\\Model\\Resolver\\SetShippingMethodsOnCart")
     addSimpleProductsToCart(input: AddSimpleProductsToCartInput): AddSimpleProductsToCartOutput @resolver(class: "Magento\\QuoteGraphQl\\Model\\Resolver\\AddSimpleProductsToCart")
+    addVirtualProductsToCart(input: AddVirtualProductsToCartInput): AddVirtualProductsToCartOutput @resolver(class: "Magento\\QuoteGraphQl\\Model\\Resolver\\AddVirtualProductsToCart")
```

Comment:
Could we reuse resolver from `addSimpleProductsToCart` (rename in order to have a more universal name) here? Resolvers for simple and virtual products are identical.

Reply:
Looks like we need to reuse configurable for simple because the code is the same (and we will have copy paste detector fail)